### ISSUE TITLE
Batch Processing

### ISSUE THREAD

**Issue by nsasanka** [Radiomics/SlicerRadiomics#53]
I have 2 questions
1. Is there a command line way to extract radiomics of one specific structure using an RT-Struct file with many contours.

2. Also is it possible to do this as a batch process for many patients using command line

Thanks

**Comment by fedorov** [comment]
No, you will need first to convert your RT-struct file into label volumes. We recommend [plastimatch](http://plastimatch.org/) for this purpose. If your RT-struct files follow the same convention for labeling individual structures, it should be relatively easy to script running plastimatch and pyradiomics in sequence for batch processing.

**Comment by timeanddoctor** [comment]
Was the code of pyradiomic installed after activation of this extension?  Then, can I run the code of pyradiomic by 'import radiomic' just as a script in slicer directly (e.g.https://www.slicer.org/wiki/Documentation/Nightly/ScriptRepository)?

**Comment by JoostJM** [comment]
Yes, installation of the radiomics package is part of the installation and is therefore available in the slicer environment if you install the extension

**Comment by timeanddoctor** [comment]
Thanks, Professor JoostJM.
I installed SlicerCaseIterator and still have some problems. The .csv table should be created including the dir of all volume and segment before loading? Can I save/write a loaded volume and mask segmentation in slicer by this module? I tried to do that but nothing happen after clicking the batch buttom.

**Comment by JoostJM** [comment]
@timeanddoctor, both volumes and segmentations loaded in slicer can be saved using the "save data" button. The default format (respectively ".nrrd" and ".seg.nrrd") are both supported in PyRadiomics.
